src/main/java/com/fasterxml/uuid/StringArgGenerator.java

```diff
@@ -16,11 +16,25 @@ public abstract class StringArgGenerator extends UUIDGenerator
      */
     public abstract UUID generate(String name);
 
+    /**
+     * Method for generating name-based UUIDs using specified name (serialized to
+     * bytes using UTF-8 encoding). No synchronization is performed on digester.
+     *  Digester is assumed to be created with ThreadLocal.
```

Comment:
nit: It will be better if we can provide an use-case example when we should use this method.